modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -15,116 +15,46 @@ images using the route host.
 
 * The following prerequisites are automatically preformed:
 ** Deploy the registry.
-** Secure the registry.
 ** Deploy a router.
 
 .Procedure
 
-. Verify whether a passthrough route exists, the route should have been created by
-default for the registry during the initial cluster installation:
-+
-----
-$ oc get route/pod-registry -o yaml
-apiVersion: v1
-kind: Route
-metadata:
-  name: podman-registry
-spec:
-  host: <host> <1>
-  to:
-    kind: Service
-    name: podman-registry <2>
-  tls:
-    termination: passthrough <3>
-----
-<1> The host for your route. You must be able to resolve this name externally by
-DNS to the router's IP address.
-<2> The service name for your registry.
-<3> Specifies this route as a passthrough route.
-+
-[NOTE]
-====
-Re-encrypt routes are also supported for exposing the secure registry.
-====
+. Expose the router using `DefaultRoute`, which is insecure:
 
-.. If it does not exist, create the route with the `oc create route passthrough`
-command, specifying the registry as the route’s service. By default, the name of
-the created route is the same as the service name:
+.. Set `DefaultRoute` to `True`.
 
-... Get the *podman-registry* service details:
+.. Log in insecurely with Podman:
 +
 ----
-$ oc get svc
-NAME              CLUSTER_IP       EXTERNAL_IP   PORT(S)                 SELECTOR                  AGE
-podman-registry   172.30.69.167    <none>        5000/TCP                podman-registry=default   4h
-kubernetes        172.30.0.1       <none>        443/TCP,53/UDP,53/TCP   <none>                    4h
-router            172.30.172.132   <none>        80/TCP                  router=router             4h
+$ podman login --insecure default-route.image-registry...
 ----
 
-... Create the route:
-+
-----
-$ oc create route passthrough    \
-    --service=podman-registry    \//<1>
-    --hostname=<host>
-route "podman-registry" created     <2>
-----
-<1> Specifies the registry as the route's service.
-<2> The route name is identical to the service name.
+////
 
-. Trust the certificates being used for the registry on your host
-system to allow the host to push and pull images. The certificates referenced
-were created when you secured your registry:
-+
-----
-$ sudo mkdir -p /etc/podman/certs.d/<host>
-$ sudo cp <ca_certificate_file> /etc/podman/certs.d/<host>
-$ sudo systemctl restart podman
-----
+You can expose the registry insecurely or securely:
+
+OR
 
-. Log in to the registry using the information from securing the registry.
-However, this time point to the host name used in the route rather than your
-service IP. When logging in to a secured and exposed registry, make sure you
-specify the registry in the `podman login` command:
+. Using Custom Routes, which is secure:
```

Comment:
We can drop "is secure" - adding your own route lets admins provide their own TLS config